<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortSetClass</class>
 <widget class="QWidget" name="SerialPortSetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPortSet</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textBrowserRecMsg"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnOpenCom">
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;串口接收到的数据：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEditSendMsg"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btnSendMsg">
     <property name="text">
      <string>发送数据</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>要发送到串口的数据：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnCloseCom">
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
